### PR TITLE
Load High Mips before Fbx after skybox

### PR DESCRIPTION
**Pull request by Atlante45**
Make sure the High Mips load as fast as possible.

## Test Plan:
1. Clear cache
2. Go to dev-welcome/dev-demo
3. The uptime of models without textures (gray models) should be greatly reduced compared to the current master.

### CODE REVIEW

File: libraries/model-networking/src/model-networking/TextureCache.cpp [hunk 1]
```diff
@@ -51,6 +51,7 @@ const std::string TextureCache::KTX_DIRNAME { "ktx_cache" };
 const std::string TextureCache::KTX_EXT { "ktx" };
 
 static const int SKYBOX_LOAD_PRIORITY { 10 }; // Make sure skybox loads first
+static const int HIGH_MIPS_LOAD_PRIORITY { 9 }; // Make sure high mips loads after skybox but before models
```
Comment (huffman) [inline libraries/model-networking/src/model-networking/TextureCache.cpp]: The resource priorities are floats, so I think we should use that here

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 526e705b0b10a346db5385e6abb89ca928614313 available at: https://gist.github.com/c15e09f3426d2e5f7295c313fb089a71<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-526e705.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-526e705.dmg

**Comment by Atlante45** [comment]
test this please

**Comment by Atlante45** [comment]
test this please

**Comment by Atlante45** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 526e705b0b10a346db5385e6abb89ca928614313 available at: https://gist.github.com/c15e09f3426d2e5f7295c313fb089a71<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-526e705.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-526e705.dmg

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
Unable to access Dev-Welcome or Dev-Demo using this build.
Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-526e705.exe

**Comment by Atlante45** [comment]
@claudioshg There was a protocol change, which is why you couldn't connect.
I updated the PR, the next build should be able to connect.

**Comment by claudioshg** [comment]
@Atlante45 Thank you for letting me know.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4f368cc867c31165fcae279eb3e7d5eb11373cb7 available at: https://gist.github.com/c15e09f3426d2e5f7295c313fb089a71<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-4f368cc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-4f368cc.dmg

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10306-4f368cc.exe

PASS:
- The domain loaded extremely fast compared to the Master. Not only did the gray models seem like they loaded immediately, but the textures loaded along with them as well.
- Tested in Dev-Welcome

**Pull request merged**